src/uu/split/src/split.rs

```diff
@@ -22,19 +22,15 @@ use std::path::Path;
 static NAME: &str = "split";
 static VERSION: &str = env!("CARGO_PKG_VERSION");
 
-mod options {
-    pub mod strategies {
-        pub static BYTES: &str = "bytes";
-        pub static LINE_BYTES: &str = "line-bytes";
-        pub static LINES: &str = "lines";
-    }
-    pub static ADDITIONAL_SUFFIX: &str = "additional-suffix";
-    pub static FILTER: &str = "filter";
-    pub static NUMERIC_SUFFIXES: &str = "numeric-suffixes";
-    pub static SUFFIX_LENGTH: &str = "suffix-length";
-    pub static DEFAULT_SUFFIX_LENGTH: usize = 2;
-    pub static VERBOSE: &str = "verbose";
-}
+static OPTION_BYTES: &str = "bytes";
```

Comment:
sorry to be a pain but s/OPTION_/OPT_/g to be consistent with the other programs :)

Reply:
oh, sorry about that... will change it